docs/sources/panels/add-a-panel.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "Add a panel"
+type = "docs"
+[menu.docs]
+identifier = "add-a-panel"
+parent = "panels"
+weight = 100
+draft = "true"
++++
+
+# Add a panel
+
+Panels allow you to show your data in visual form. This topic walks you through the most basic steps to build a panel.
+
+## 1. Add a panel to a dashboard
+
+1. Navigate to the dashboard you want to add a panel to.
+1. Click the **Add panel** icon.
+   
+     ![](/img/docs/panels/add-panel-icon-7-0.png)
+
+1. Click **Add new panel**.
+
+Grafana creates an empty graph panel with your default data source selected.
+
+## 2. Edit panel settings
+
+While not required, we recommend that you add a helpful title and description to your panel. You can use variables you have defined in either field, but not global variables.
```

Comment:
thi
```suggestion
While not required, we recommend that you add a helpful title and description to your panel. You can use template variables you have defined in your dashboard.
```

Reply:
We don't actually use the term "template variables" anywhere in the UI, and while people like you and Daniel seem to know what they are, I am not confident that anyone else does. The global variables are listed and defined in the Global variables topic. I will add some clarifying links.

"Template" as defined to me (and in the docs) is any text that includes a variable. So "template variable" is not at all clear.